agent/php_globals.h

```diff
@@ -80,6 +80,7 @@ typedef struct _nrphpglobals_t {
                                               enabled */
   nr_composer_api_status_t composer_api_status; /* Composer API's status */
   char* docker_id; /* 64 byte hex docker ID parsed from /proc/self/mountinfo */
+  bool is_horizon_used; /* Set to true if Laravel Horizon is used */
```

Comment:
I suggest adding more context to the name of this global: `laravel_horizon_worker_used`.

Reply:
Good point, changed: [1604a0e](https://github.com/newrelic/newrelic-php-agent/pull/1132/commits/1604a0e718dff79290cecf3c523312c6c7d40702)